py/bitbox02/bitbox02/util.py

```diff
@@ -141,3 +144,68 @@ def set_app_static_privkey(self, privkey: bytes) -> None:
         data = self._read_cache()
         data.app_static_privkey = privkey
         self._write_cache(data)
+
+
+class BitBoxAppNoiseConfig(bitbox_api_protocol.BitBoxNoiseConfig):
+    """
+    Noise config that reads and stores the noise keys in the same location as the BitBoxApp.
```

Comment:
I think it would be good to clarify the purpose of `NoiseConfigUserCache` in its docstring. If I were an integrating app which uses this package, I wouldn't know for sure which one to use: this or NoiseConfigUserCache or both.

Reply:
Done.

Ideally you should use none though, but your custom one ;) (also e.g. to implement the attestation check).

Reply:
Well, I could still subclass `BitBoxAppNoiseConfig`, right, which would at least give me the noise config.